waterloop-site/src/components/RecruitmentForm/JobPostingPage.tsx

```diff
@@ -1,29 +1,38 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import JobPosting from './JobPosting';
-import { w21Postings } from '../../static/consts/postings';
-import { useParams } from 'react-router';
+import moment from 'moment';
+import { useParams, useHistory } from 'react-router';
+import usePostingPostingById from 'hooks/posting-by-id';
 
 interface RouteParams {
   id: string;
 }
+const applicationForm: string = '';
```

Comment:
This being a constant implies its value not changing elsewhere in the code, so is this meant to be hard coded?